package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "ng": "ng",
     "build:staging": "npm run codegen:gql && ng build --c staging --base-href=https://catcher-org.github.io/WATcher-staging/",
-    "build:prod:web": "npm run codegen:gql && ng build --prod --base-href=https://CATcher-org.github.io/WATcher/",
+    "build:prod:web": "npm run codegen:gql && ng build --prod --base-href=https://catcher-org.github.io/WATcher/",
```

Comment:
As this in unrelated to the current PR, could you create an issue and make this change in a separate PR?